website/docs/copa-generate.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Copa Generate Command
```

Comment:
we should have a more explanatory name for this,  something like `Docker Input Integration`? open to suggestions